build.gradle

```diff
@@ -75,7 +75,7 @@ configure(subprojects.findAll { it.name != 'util' }) {
         options.encoding = 'ISO-8859-1'
         options.fork = true
         options.debug = true
-        options.compilerArgs = ['-Xlint:all', '-Xlint:-options']
+        options.compilerArgs = ['-Xlint:all', '-Xlint:-options', '-Xlint:-deprecation']
```

Comment:
:(   But otherwise we have to fully qualify references to every deprecated API element in core

Reply:
Can we add a todo or ticket to remove in 4.0?